Jellyfin.Server.Implementations/StorageHelpers/StorageHelper.cs

```diff
@@ -72,6 +72,11 @@ public static FolderStorageInfo GetFreeSpaceOf(string path)
     private static void TestDataDirectorySize(string path, ILogger logger, long threshold = -1)
     {
         logger.LogDebug("Check path {TestPath} for storage capacity", path);
+        if (Directory.Exists(path))
```

Comment:
```suggestion
        if (!Directory.Exists(path))
```

Should this be inverted, if it doesn't exist create it ?